config.php

```diff
@@ -251,6 +251,9 @@
     if ( ! defined( 'WP_FS__GDPR_OPTION_NAME' ) ) {
         define( 'WP_FS__GDPR_OPTION_NAME', WP_FS___OPTION_PREFIX . 'gdpr' );
     }
+    if ( ! defined( 'WP_FS__CLONE_MANAGEMENT_OPTION_NAME' ) ) {
```

Comment:
@fajardoleo the define is only used in the clone manager, and only once, so I think there's no need to have a "define" for it.